tests/e2e/upgrade/upgrade/check-jaeger-version.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ "$#" -ne 3 ]; then
+    echo "$0 <Jaeger name> <namespace> <expected version>"
+    exit 1
+fi
+
+check_version() {
+    export VERSION=$(kubectl get jaeger $jaeger -o jsonpath="{.status.version}" -n $namespace)
+}
+
+
+jaeger=$1
+namespace=$2
+expected_version=$3
+
+check_version
+while [ "$VERSION" != "$expected_version" ]
```

Comment:
is this going to while forever if the version doesnt matdh?

Reply:
Yes, but KUTTL timeout will finish the execution of this script